src/python_testing/TC_CADMIN_1_22_24.py

```diff
@@ -0,0 +1,179 @@
+        sleep(1)
+
+        self.step(4)
+        window_status = await self.get_window_status()
+        asserts.assert_equal(window_status, Clusters.AdministratorCommissioning.Enums.CommissioningWindowStatusEnum.kWindowNotOpen,
+                             "Commissioning window is expected to be closed, but was found to be open")
+
+        self.step(5)
+        try:
+            await self.th1.OpenCommissioningWindow(
+                nodeid=self.dut_node_id, timeout=901, iteration=10000, discriminator=self.discriminator, option=1)
+
+        except ChipStackError as e:
+            # Since we provided 901 seconds as the timeout duration,
+            # we should not be able to open comm window as duration is too long.
+            # we are expected receive Failed to open commissioning window: IM Error 0x00000585: General error: 0x85 (INVALID_COMMAND)
+            _INVALID_COMMAND = 0x00000585
+            asserts.assert_equal(e.err,  _INVALID_COMMAND,
+                                 "Expected to error as we provided failure value for opening commissioning window")
+
+        self.step(6)
+        window_status2 = await self.get_window_status()
+        asserts.assert_equal(window_status2, Clusters.AdministratorCommissioning.Enums.CommissioningWindowStatusEnum.kWindowNotOpen,
+                             "Commissioning window is expected to be closed, but was found to be open")
+
+    def pics_TC_CADMIN_1_24(self) -> list[str]:
+        return ["CADMIN.S"]
+
+    def steps_TC_CADMIN_1_24(self) -> list[TestStep]:
+        return [
+            TestStep(1, "Commissioning, already done", is_commissioning=True),
+            TestStep(2, "TH_CR1 opens a commissioning window on DUT_CE using ECM with a value of 180 seconds",
+                     "DUT_CE opens its Commissioning window to allow a second commissioning"),
+            TestStep(3, "Wait for the commissioning window in step 2 to timeout"),
+            TestStep(4, "TH_CR1 reads the window status to verify the DUT_CE window is closed",
+                     "DUT_CE windows status shows the window is closed"),
+            TestStep(5, "TH_CR1 opens a commissioning window on DUT_CE using ECM with a value of 179 seconds",
+                     "DUT_CE does not open its Commissioning window to allow a second commissioning. DUT_CE shows 'Failed to open commissioning window. Global status 0x85'"),
+            TestStep(6, "TH_CR1 reads the window status to verify the DUT_CE window is closed",
+                     "DUT_CE windows status shows the window is closed"),
+        ]
+
+    @async_test_body
+    async def test_TC_CADMIN_1_24(self):
+        self.step(1)
+        self.th1 = self.default_controller
+        self.discriminator = random.randint(0, 4095)
+
+        self.step(2)
+        await self.th1.OpenCommissioningWindow(
+            nodeid=self.dut_node_id, timeout=180, iteration=10000, discriminator=self.discriminator, option=1)
+
+        self.step(3)
+        sleep(180)
```

Comment:
yaml used to wait for 181 seconds instead of 180 for some reason.

Also this is absolutely terrible for CI. Could we make CI sleep less? Or maybe have an interface to skip slow in PR CI the same like we have for YAML: https://github.com/project-chip/connectedhomeip/blob/master/scripts/tests/chiptest/__init__.py#L98

Reply:
I apologize for this test having caused some issues in the CI currently due to the sleep time.
I will file a PR here soon to remove this test from running in the CI moving forward, sounds like this test is not needed to be run in the CI.

Reply:
Hi @andy31415, 
I noticed that there is some new code in src/python_testing/test_metadata.yaml in the master branch.
In the slow_tests section of that yaml file there appears to be CADMIN_1_9, CADMIN_1_19, and CADMIN_1_22_24 tests with duration of test runs added currently. 
Does this resolve the issue with the CI and the test from this above PR? Can this comment be resolved now?

Should the CADMIN_1_22_24 python test module also be added to the "not_automated" section with a caveat that it is causing issues due to long run time in order for it to not be selected in the execute_python_tests.py script during test runs in CI by the test runners?